tests/integration/test_utils/data_source_config/pandas_filesystem_csv.py

```diff
@@ -0,0 +1,71 @@
+import pathlib
+
+import pandas as pd
+import pytest
+
+from great_expectations.compatibility.typing_extensions import override
+from great_expectations.datasource.fluent.interfaces import Batch
+from tests.integration.test_utils.data_source_config.base import (
+    BatchTestSetup,
+    DataSourceTestConfig,
+)
+
+
+class PandasFilesystemCsvDatasourceTestConfig(DataSourceTestConfig):
+    @property
+    @override
+    def label(self) -> str:
+        return "pandas-filesystem-csv"
+
+    @property
+    @override
+    def pytest_mark(self) -> pytest.MarkDecorator:
+        return pytest.mark.filesystem
+
+    @override
+    def create_batch_setup(
+        self, data: pd.DataFrame, request: pytest.FixtureRequest
+    ) -> BatchTestSetup:
+        tmp_path = request.getfixturevalue("tmp_path")
```

Comment:
Hmm this feels a little magical but I guess there's no way around it?

Reply:
Yeah, I don't love it either, but it's the best I could come up with. Alternatives I considered:
* have `batch_for_datasource` (the pytest.fixture) have `tmp_path` as one of the fixtures in its parameters, and then pass that to `create_batch_setup`: I decided against this because it's not needed for many data sources, and introducing that pattern could lead us to later passing a bunch of stuff into `create_batch_setup`
* not using the fixture, and just doing the temp path generation ourselves. Seems like the additional code wasn't worth it to me